Samples/ResourceManagement/Common/Utilities.cs

```diff
@@ -22,6 +22,23 @@ namespace Microsoft.Azure.Management.Samples.Common
 {
     public static class Utilities
     {
+        public static Action<string> LoggerMethod { get; set; }
+
+        static Utilities()
+        {
+            LoggerMethod = Utilities.Log;
+        }
+
+        public static void Log(string message)
+        {
+            LoggerMethod.Invoke(message);
+        }
```

Comment:
Something seems missing.
LoggerMethod is Utilities.Log which is again calling LoggerMethod. are you able to print anything will this. It seems to be recursive call. Also in tests/Sample method should be assigned to something. I would say by default it should be Console.Writeline but in tests we can just override it